convert_file.py

```diff
@@ -0,0 +1,91 @@
+import sys
+import argparse
+import gzip
+import shutil
+import tarfile
+import patoolib
+
+
+class FilesFamily(object):
+
+    def __init__(self, list_of_formats):
+
+        # List of formats that the class supports, represented by
```

Comment:
You should use the function docstring to document the parameter's purpose
It should look like this
```python
     def __init__(self, list_of_formats):
          """
          Args:
               list_of_formats (list[str]): List of formats that the.....
          """
```
You can see more examples [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)